packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts

```diff
@@ -320,10 +331,318 @@ export class SchemaCRUDPlugin extends GraphbackPlugin {
           });
         }
       }
-
     });
   }
 
+  /**
+   * Create resolvers function that
+   *
+   * @param inputContext
+   * @param options
+   */
+  protected transformSchemaToString(schema: GraphQLSchema) {
+    const schemaString = printSchemaWithDirectives(schema);
+    if (this.pluginConfig) {
+      if (this.pluginConfig.format === 'ts') {
+        return tsSchemaFormatter.format(schemaString)
+      }
+      if (this.pluginConfig.format === 'js') {
+        return jsSchemaFormatter.format(schemaString)
+      }
+      if (this.pluginConfig.format === 'graphql') {
+        return gqlSchemaFormatter.format(schemaString)
+      }
+    }
+    throw Error("Invalid format specified. `options.format` supports only `ts`, `js` and `graphql` flags");
+  }
+
+  /**
+   * Create Query resolver fields
+   *
+   * @param {ModelDefinition} model - The model definition with CRUD config and GraphQL typr
+   * @param {IFieldResolver} queryObj - Query resolver object
+   */
+  protected addQueryResolvers(model: ModelDefinition, queryObj: any) {
```

Comment:
```suggestion
  protected addQueryResolvers(model: ModelDefinition, queryObj: IFieldResolver) {
```

Not sure how accurate the typing is just copied it from method comments. So it will need verification

Reply:
Also, it has to be applied throughout

Reply:
Yep wanted to come back to this but wanted to push before my power went (planned power outage here today) - it is back now.